db/changes.go

```diff
@@ -170,10 +170,10 @@ func (db *Database) AddDocInstanceToChangeEntry(entry *ChangeEntry, doc *Documen
 
 // Creates a Go-channel of all the changes made on a channel.
 // Does NOT handle the Wait option. Does NOT check authorization.
-func (db *Database) changesFeed(singleChannelCache SingleChannelCache, options ChangesOptions, currentCachedSequence uint64, to string) (<-chan *ChangeEntry, error) {
+func (db *Database) changesFeed(changesFeedOutput chan *ChangeEntry, singleChannelCache SingleChannelCache, options ChangesOptions, currentCachedSequence uint64, to string) (<-chan *ChangeEntry, error) {
```

Comment:
I think we can revert the change to pass changesFeedOutput down now that we're just using FatalPanicHandler.